### DISCUSSION TITLE
ToC on the left for README

### DISCUSSION THREAD

**Discussion by demberto** [squidfunk/mkdocs-material#4183] [Q&A]
The main `README` of my docs, has a single element in the navigation section. I would like to hide the navigation and put TOC to the left, how?

**Comment by squidfunk** [comment]
You can use the [`toc.integrate`](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/?h=toc+integrate#navigation-integration) feature.

**Reply by demberto** [reply]
I want to do it only for a single page, not the entire docs

**Reply by squidfunk** [reply]
Definitely possible with CSS, but this would require significant effort.
